.github/workflows/scripts/run_unit_tests.sh

```diff
@@ -14,3 +14,9 @@ if [ -n "${COVERALLS_TOKEN}" ]; then
     "$(go env GOPATH)"/bin/goveralls -coverprofile="${COVERPROFILE}" \
             -service=github
 fi
+
+# re-run with race detector
+echo
+echo "Re-running with race detector enabled, skipping flaky tests..."
+echo
+GOVERBOSE=1 make race-test
```

Comment:
Is there something normally lacking in the output wherein we'd want to force `-v` for CI/CD?

Reply:
Initially, just for debugging. I'm thinking of leveraging `t.Log` to report out the issues with the enhancement to require that the conditional skip must have an issue associated. Without verbose, those logs won't be visible in the CI/CD runs.